cmd/job_run.go

```diff
@@ -69,29 +70,45 @@ func (cmd *JobRun) Execute(args []string) (err error) {
 
 	//start with input volumes
 	//@TODO move this logic to a separate package and test is
-	var inputDataset string
+	var inputDatasets []string
 	vols := map[string]*svc.JobVolume{}
 	for _, input := range cmd.Inputs {
```

Comment:
Could you take a look at the possibility of moving these path "parsing" and assertion logic to a separate function and add unit tests? Seems perfect for table testing: [https://blog.golang.org/subtests](https://blog.golang.org/subtests)!